packages/host/app/components/operator-mode/profile/profile-subscription.gts

```diff
@@ -0,0 +1,197 @@
+import { fn } from '@ember/helper';
+
+import { on } from '@ember/modifier';
+import Owner from '@ember/owner';
+import { service } from '@ember/service';
+import Component from '@glimmer/component';
+
+import { restartableTask, task } from 'ember-concurrency';
+
+import perform from 'ember-concurrency/helpers/perform';
+import window from 'ember-window-mock';
+
+import {
+  BoxelButton,
+  FieldContainer,
+  LoadingIndicator,
+} from '@cardstack/boxel-ui/components';
+import { or } from '@cardstack/boxel-ui/helpers';
+import { IconHexagon } from '@cardstack/boxel-ui/icons';
+
+import WithSubscriptionData from '@cardstack/host/components/with-subscription-data';
+import BillingService from '@cardstack/host/services/billing-service';
+
+interface Signature {
+  Args: {};
+  Element: HTMLElement;
+}
+
+export default class ProfileSubscription extends Component<Signature> {
+  <template>
+    <WithSubscriptionData as |subscriptionData|>
+      <FieldContainer
+        @label='Membership Tier'
+        @tag='label'
+        class='profile-field'
+      >
+        <div class='profile-subscription'>
+          <div class='monthly-credit'>
+            <div class='plan-name'>{{subscriptionData.plan}}</div>
+            <div class='credit-info'>
+              <span class='credit-info__label'>Monthly Credit</span>
+              {{subscriptionData.monthlyCredit}}
+            </div>
+          </div>
+          <BoxelButton
+            @kind='secondary-light'
+            @size='extra-small'
+            @disabled={{or
+              subscriptionData.isLoading
+              this.fetchStripeLinks.isRunning
+              this.managePlan.isRunning
```

Comment:
I find this odd, why would `managePlan` be "running" if it's simply an external stripe link?